content/courses/tokens-and-nfts/nfts-with-metaplex.md

```diff
@@ -19,8 +19,8 @@ description:
   files, any special traits the NFT has, and more.
 - The **Metaplex Token Metadata** program is an onchain program that attaches
   metadata to a token mint. We can interact with the Token Metadata program
-  using the `metaplex-foundation/js` npm module, also known as the
-  [Metaplex JavaScript SDK](https://github.com/metaplex-foundation/js).
+  using UMI and the
+  [Token Metadata plugin](https://developers.metaplex.com/token-metadata).
```

Comment:
Can you change this PR to focus on `mpl-token-metadata` rather than `Umi`.  People searching for Umi get some [very complicated docs that won't help them make tokens](https://developers.metaplex.com/umi) but if we tell them they're using the `mpl-token-metadata` program they'll get the [better docs yu mentioned](https://developers.metaplex.com/token-metadata). 

Also Umi is in lowercase.